docs/reference/user/connection.md

```diff
@@ -1,36 +1,22 @@
 # Connection {#RefConnection}
 
-Connections between users are a means for giving users some control over their
-privacy and a degree of spam protection. Connections control whether a user can:
+A connection is a _directed_ edge from one user to another that is attributed with a relation state and some meta information. If a user has a connection to another user, it can be in one of the six [connection states](#RefConnectionStates).
```

Comment:
Hrmm; this may be true as an implementation detail; but maybe it muddies the semantics of what it means to be *connected*? Users should never observe the *directedness* of the connection right?

If that's the case it may be worth mentioning that this is only how it's represented internally.

Reply:
> Users should never observe the directedness of the connection right?

I haven't realized it but it seems to be true, at least where it matters ([`ensureConnected`](https://github.com/wireapp/wire-server/blob/artyom-registration/services/galley/src/Galley/API/Util.hs#L43-L55)).

I'll update the doc.